plugins/network-elements/netscaler/src/com/cloud/network/element/NetscalerElement.java

```diff
@@ -260,7 +264,7 @@ public boolean applyLBRules(Network config, List<LoadBalancingRule> rules) throw
         Map<Capability, String> lbCapabilities = new HashMap<Capability, String>();
 
         // Specifies that the RoundRobin and Leastconn algorithms are supported for load balancing rules
-        lbCapabilities.put(Capability.SupportedLBAlgorithms, "roundrobin,leastconn");
+        lbCapabilities.put(Capability.SupportedLBAlgorithms, "roundrobin,leastconn,source");
```

Comment:
add space after "," 
should be like lbCapabilities.put(Capability.SupportedLBAlgorithms, "roundrobin,leastconn, source")